buildbot-host/rebuild-all.sh

```diff
@@ -18,3 +18,5 @@ done
 
 # tends to produce several dangling images
 docker image prune --force
+# clean up build cache
+docker system prune --force
```

Comment:
`system prune` includes `image prune` - would it make sense to just replace `image` with `system`?